src/sentry/static/sentry/app/utils/performance/quickTrace/traceLiteQuery.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import GenericDiscoverQuery from 'app/utils/discover/genericDiscoverQuery';
+import {
+  TraceLite,
+  TraceLiteQueryChildrenProps,
+  TraceProps,
+  TraceRequestProps,
+} from 'app/utils/performance/quickTrace/types';
+import {
+  beforeFetch,
+  getQuickTraceRequestPayload,
+  isTransaction,
+  makeEventView,
+} from 'app/utils/performance/quickTrace/utils';
+import withApi from 'app/utils/withApi';
+
+type QueryProps = Omit<TraceRequestProps, 'eventView'> & {
+  children: (props: TraceLiteQueryChildrenProps) => React.ReactNode;
+};
+
+function getQuickTraceLiteRequestPayload({event, ...props}: TraceRequestProps) {
+  const additionalApiPayload = getQuickTraceRequestPayload(props);
+  return Object.assign({event_id: event.id}, additionalApiPayload);
+}
+
+function EmptyTrace({children}: Pick<QueryProps, 'children'>) {
+  return (
+    <React.Fragment>
+      {children({
+        isLoading: false,
+        error: null,
+        trace: null,
+        type: 'partial',
```

Comment:
Should this be empty trace type?

```suggestion
        type: 'empty',
```

Reply:
That's a good question. I intended this to be used to differentiate the results between the lite and full queries, and empty was just useful to indicate neither was used. But I see your point about empty here.

For this PR I don't think it's important what these values are set to but it may be useful in the future to set this as empty. 👍

Reply:
Scratch that. The full and lite queries actually return different types. This means that there isn't an empty type that works for both without complicating everything. I'm going to keep this `type` just to differentiate where the results came from and only add the empty type to the quick trace query which converts the results of a full trace query into a flat list.